Makefile

```diff
@@ -28,6 +28,10 @@ specs/swagger-download-patch.json: specs/swagger-stripped-oauth.json
 	# We add missing binary response body spec to the swagger
 	jq '.definitions."domain.DownloadItem"."type"="string" | .definitions."domain.DownloadItem"."format"="binary"' $< > $@
 
+specs/swagger-pdf-download-patch.json: specs/swagger-download-patch.json
```

Comment:
I might be reading this wrong, but does `specs/swagger-download-patch.json` no longer exist?